### ISSUE TITLE
PRI2: Logika hledání např. v Hlídač veřejných zakázek

### ISSUE THREAD

**Issue by MartinKamin** [HlidacStatu/Hlidac-Statu#391]
Při otevření stránky Veřejných zakázek má uživatel DVĚ vyhledávací okna.
Do prvního funguje vepisování pomocí <slovo>ENTER<slovo> Hledat
Do druhého Enter nefunguje a hledání se zadává jako čistý textových string.
![image](https://github.com/user-attachments/assets/8752057b-56a7-4612-acad-829359b7c650)

Myslím, že by mělo být jedno okno na vyhledávání a také by mělo vkládání fungovat stejně jako globální
[https://www.hlidacstatu.cz/verejnezakazky/hledat?Q=icozadavatel%3A00000078+AND+icododavatel%3A27392244&Oblast=&Zahajeny=false](
https://www.hlidacstatu.cz/verejnezakazky/hledat?Q=icozadavatel%3A00000078+AND+icododavatel%3A27392244&Oblast=&Zahajeny=false)

**Comment by suchoss** [comment]
@MartinKamin Jo, jenže tu řešíme "problém" jak to udělat? Zatím nemám vymyšlené řešení. To jedno vyhledávací okno vyhledává vždy všude. To druhé vyhledávací okno vyhledává jen ve VZ. Pokud máš nápad, jak to udělat, tak sem s ním.

**Comment by suchoss** [comment]
Bude fixnuto jinak
